etcdserver/api/membership/cluster_test.go

```diff
@@ -461,7 +461,7 @@ func TestClusterAddMember(t *testing.T) {
 			Params: []interface{}{
 				path.Join(StoreMembersPrefix, "1", "raftAttributes"),
 				false,
-				`{"peerURLs":null}`,
+				`{"peerURLs":null,"isLearner":false}`,
```

Comment:
If isLeadner is absent, does it default to false? Test both cases (where "isLearner" is false, and when it's absent)?

Reply:
This existing test does not explicitly set isLearner field, and the field is defaulted to false. It is a good idea to have a test with isLearner absent, set to false, and set to true. I will add it.